src/Components/NavGenre.js

```diff
@@ -11,16 +17,40 @@ const NavGenre = () => {
         </LogoLink>
       </Logo>
       <GenreList>
-        <Link>홈</Link>
+        <Listcusor
+          onClick={() => {
+            changePage("/");
+          }}
```

Comment:
```suggestion
          onClick={() => changePage("/")}
```
이렇게 하는게 어떨까요? 아니면 goToMain 같은 함수로 아예 빼주면 어떨까요?

Reply:
Main 탭 말고 다른 장르로 가는 부분도 있는데 이 부분도 전부 함수로 바꿔주는게 좋을까요 ?